ci/code_checks.sh

```diff
@@ -22,6 +23,7 @@ fi
 if [ "${skipexpensive:-}" != "true" ]; then
   echo "Type checking"
   pytype -j auto "${SRC_FILES[@]}"
+  mypy --show-error-codes "${SRC_FILES[@]}"
```

Comment:
Yes, but it incentivizes users to fix those errors? Since code_checks.sh is only for local development. 

To be fair, I guess you're right and we should just open a PR for that (or for each file) as I don't think anyone will actually read the error messages.

We will replace it with precommit / makefile soon anyways.